syntax = "proto3";
package hot_load;

service HotLoadModelService {
    rpc loading(Request) returns (Response) {}
}

message Request {

  string model_name = 1;

  string model_file_address = 2;

  bool is_remote = 3;

  bool is_tar_packed = 4;

  int32 timeout = 5;

  string attach = 6;
};

message Response {

  int32 err_no = 1;

  string err_msg = 2;
};
